### DISCUSSION TITLE
--ignore-unused-services switch not working

### DISCUSSION THREAD

**Discussion by sr-rossanderson** [prowler-cloud/prowler#2972] [Q&A] [answered]
When I try to use the --ignore-unused-services switch with my aws scan I get an "unrecognized arguments" error.

I'm running version 3.10.0

**Comment by jfagoagas** [comment, marked as answer]
Hi @neueda-randerson, the `--ignore-unused-services` flag and feature will be available in the next Prowler release which will be `3.11.0`. If you want to use it before the release you can use the `latest` container or the `master` branch in Github but those are not stable.

Thanks for using Prowler 🚀

**Reply by sr-rossanderson** [reply]
Apologies - hadn't realised this was not released as yet.

Thanks for coming back to me
